### PR TITLE
[BACKLOG-24402] Adds more styling to Kettle status page and fixes issue with styles displaying on Linux

### PR DESCRIPTION
**Pull request by pentaho-mracine**

### CODE REVIEW

**Comment by pentaho-mracine** [comment]
@pentaho/r2d2 Please review

File: engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java [hunk 1]
```diff
@@ -253,74 +251,78 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
...
-          line = mantleBr.readLine();
-          while ( line != null ) {
-            sb.append( line );
-            line = mantleBr.readLine();
-          }
-
-          styleBr.close();
-          out.println( sb.toString() );
-          out.println( "</STYLE>" );
+          out.println( "<link rel=\"stylesheet\" type=\"text/css\" href=\"/pentaho/content/common-ui/resources/themes/" + themeName + "/" + themeCss + "\"/>" );
```
Comment (bmorrise) [inline engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: How will the styles be handle in the context of standalone carte when pentaho server is not available?

Comment (pentaho-mracine) [reply engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: I guess this code doesn't account for that, but should a standalone carte even display themes? If so I would think we'd need to copy all of the Pentaho themes located in the pentaho-solutions/system/common-ui folder over to PDI and then add more logic into this to look into both locations.

**Comment by pentaho-mracine** [comment]
Closing until we can have the themes more accessible from PDI.

**Comment by pentaho-mracine** [comment]
Reopening as we will tackle the standalone carte server styling later.

**Pull request merged**